generated/Contract/Contract.ts

```diff
@@ -10,68 +10,757 @@ import {
   BigInt
 } from "@graphprotocol/graph-ts";
 
-export class AdminChanged extends ethereum.Event {
-  get params(): AdminChanged__Params {
-    return new AdminChanged__Params(this);
+export class Approval extends ethereum.Event {
```

Comment:
Why do we need this file?

Reply:
deleted the old generated and build, the new one removed all of these events